include/seqan3/alignment/multiple/align_multiple.hpp

```diff
@@ -0,0 +1,95 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides the algorithm seqan3::align_multiple for multiple sequence alignment.
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ * \author Joerg Winkler <j.winkler AT fu-berlin.de>
+ * \author Simon Sasse <simon.sasse AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/std/ranges>
+#include <vector>
+
+#include <seqan3/core/platform.hpp>
+
+#if SEQAN3_HAS_SEQAN2
+#include <seqan/graph_msa.h>
+#endif
+
+#include <seqan3/alignment/configuration/align_config_gap.hpp>
+#include <seqan3/alignment/multiple/detail/align_multiple_seqan2_adaptation.hpp>
+
+namespace seqan3::align_cfg
+{
+
+/*!\brief The standard configuration for multiple sequence alignment.
+ * \ingroup multiple_alignment
+ * \details
+ *
+ * The standard configuration values are adapted from SeqAn2 in order to provide the same behaviour.
+ *
+ *   * general gap score: -1
+ *   * additional gap open score: -13
+ *   * band constraints: none
+ *   * scoring for amino acid sequences: Blosum62 matrix (set internally)
+ *   * scoring for nucleotide sequences: match = +5 and mismatch = -4 (set internally)
+ *
+ * \if DEV
+ * See seqan3::detail::align_multiple_seqan2_adaptation::create_msa_configuration for more details on the configuration.
+ * \endif
+ */
+constexpr configuration msa_default_configuration = gap{gap_scheme{gap_score{-1}, gap_open_score{-13}}};
```

Comment:
I think this is not required. The adaptor already selects the defaults and we can simply use an empty seqan3::configration as the optinal argument if that is ok with the algorithm.

Reply:
I thought this makes the default (that I can configure) more explicit but sure I can move this into the adaptions.